src/replication.c

```diff
@@ -3606,6 +3606,7 @@ void unblockClientWaitingReplicas(client *c) {
     listNode *ln = listSearchKey(server.clients_waiting_acks,c);
     serverAssert(ln != NULL);
     listDelNode(server.clients_waiting_acks,ln);
+    updateStatsOnUnblock(c, 0, 0, 0);
```

Comment:
I know I suggested it, but this makes the wait case counted twice - we should now remove the additions in replyToBlockedClientTimedOut